packages/store/ts/codegen/renderTable.ts

```diff
@@ -104,9 +112,9 @@ library ${libraryName} {
   ${renderWithStore(
     storeArgument,
     (_typedStore, _store, _commentSuffix) => `
-    /** Register the table's key schema, value schema, key names and value names${_commentSuffix} */
+    /** Register the table keys' and values' field layout, key names and value names${_commentSuffix} */
```

Comment:
I don't think field layout applies to keys, so I went to adjust this comment, but then realized maybe we don't need to include all of this method's implementation in the comment? Maybe just calling out "its config" is enough?

```suggestion
    /** Register the table with its config${_commentSuffix} */
```